Ghidra/Processors/8048/data/languages/8048.cspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<compiler_spec>
+  <global>
+    <range space="CODE"/>
+    <range space="INTMEM"/>
+    <range space="EXTMEM"/>
+    <range space="PORT"/>
+  </global>
+  <stackpointer register="SP" space="INTMEM" growth="positive"/>
+  <returnaddress>
+    <varnode space="stack" offset="-2" size="2"/>
+  </returnaddress>
+  <default_proto>
+    <prototype name="__stdcall" extrapop="-2" stackshift="-2" strategy="register">
+      <input>
+        <pentry minsize="1" maxsize="1">
+          <register name="A"/>
+        </pentry>
+      </input>
+      <output>
+        <pentry minsize="1" maxsize="1">
+          <register name="A"/>
+        </pentry>
+      </output>
+      <unaffected>
+        <register name="SP"/>
+      </unaffected>
+      <localrange>
```

Comment:
The local offset values are relative to the stack pointer.  The start should be zero.  These lines should really be removed unless it was causing issues without them.
As is, the local variables won't start until SP+8.